Tools/Scripts/webkitpy/layout_tests/controllers/test_result_writer.py

```diff
@@ -100,18 +100,22 @@ def output_filename(self, modifier):
 
         ext_parts = fs.splitext(self._test_name)
         output_basename = ext_parts[0]
+        extension = ext_parts[1]
```

Comment:
What is the guarantee that ext_parts has more than 1 entry?

Reply:
It's an API guarantee from Python, see https://docs.python.org/3/library/os.path.html?highlight=os%20path%20splitext#os.path.splitext